Examples/Io/Root/include/ActsExamples/Io/Root/RootTrackStatesWriter.hpp

```diff
@@ -89,7 +89,17 @@ class RootTrackStatesWriter final : public WriterT<ConstTrackContainer> {
                      const ConstTrackContainer& tracks) override;
 
  private:
-  enum ParameterType { ePredicted, eFiltered, eSmoothed, eUnbiased, eSize };
+  enum ParameterType { ePredicted = 0, eFiltered, eSmoothed, eUnbiased, eSize };
+  enum StateType {
```

Comment:
Nitpick
```suggestion
  enum struct StateType {
```

since you're already doing a static cast below.